app-frontend/src/app/services/common/decimal.filter.js

```diff
@@ -0,0 +1,13 @@
+export default (app) => {
+    app.filter('setDecimal', function () {
+        'ngInject';
+        return function (input, places) {
+            if (isNaN(input)) {
+                return input;
+            }
+
+            let factor = '1' + Array(+(places > 0 && places + 1)).join('0');
```

Comment:
This is slick. But I would definitely add a comment here with an example input and output of this expression, because it definitely takes a few seconds to figure out what's happening.